src/main/java/org/pucko/commands/Cd.java

```diff
@@ -1,43 +1,64 @@
 package org.pucko.commands;
 
+import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.ArrayList;
 
 import org.pucko.core.WorkingDirectory;
 
-public class Cd {
-    private ArrayList<String> args;
-    private WorkingDirectory wd;
+public class Cd extends Command {
 
 
     public Cd(ArrayList<String> args, WorkingDirectory wd) {
```

Comment:
Hm, nä jag tycker inte det behövs. Då måste ju grep heta:
globally search a regular expression and print